rmgpy/data/kinetics/family.py

```diff
@@ -2139,6 +2139,10 @@ def assignWeightsToEntries(entryNestedList, weightedEntries, N = 1):
             training = []
             
             lines = kinetics.comment.split('\n')
+            
+            euc = [i for i in xrange(len(lines)) if lines[i].startswith('Euclid')] #remove the Euclidean distance line to help parser
```

Comment:
Is this just `euc = [line for line in lines if line.startswith('Euclid')]` ?

Or the two lines together:
`lines = [line for line in lines if not line.startswith('Euclid')]`
?